playground/src/PlaygroundView.tsx

```diff
@@ -67,21 +82,26 @@ export class PlaygroundView extends React.Component<IPlaygroundViewProps, IPlayg
   }
 
   public render(): React.ReactNode {
-    const mainRowStyle: React.CSSProperties = {
-      alignItems: 'stretch',
-      flex: 1
-    };
-
-    const errorsPaneStyle: React.CSSProperties = {
-      height: '130px',
-      marginTop: '20px'
-    };
+    return (
+      <div>
+        <Media query="(min-width: 700px)">
+          {
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
+            (matches: any) =>
+              matches
+                ? this._renderLargeScreenPlayground() // For example: Laptops
+                : this._renderSmallScreenPlayground() // For example: iPhones/iPads
+          }
+        </Media>
+      </div>
```

Comment:
Why didn't you use simple CSS media queries?

Reply:
Hello, thank you for the feedback.
I guess more of a personal preference, since I used media query like this in past projects. Would you like me to switch away from this?

Reply:
It is more efficient to do such things with CSS than with JS (IMO)

Reply:
Awesome, made changes to use CSS rather than JS. Thank you!